### PR TITLE
4657 - Integration test (penalties flow)

### PR DESCRIPTION
**Pull request by matthopson**
Tests penalty input flow

Screen Shot 2020-05-20 at 3 43 55 PM
Screen Shot 2020-05-20 at 3 44 03 PM
Screen Shot 2020-05-20 at 3 44 17 PM
Screen Shot 2020-05-20 at 3 44 59 PM
Screen Shot 2020-05-20 at 3 44 40 PM
Screen Shot 2020-05-20 at 3 44 28 PM

### CODE REVIEW

File: web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        value: '$1,000',
+      });
+
+      await test.runSequence('updateFormValueSequence', {
+        key: `statistics.${i}.totalPenalties`,
+        value: '$100',
+      });
+    }
+
+    //await test.runSequence('saveSavedCaseForLaterSequence');
```
Comment (rachaelparris) [inline web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js]: Is this commented out on purpose?

File: web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    // Fill out all statistics values and submit
+    for (let i = 0; i < 12; i++) {
+      await test.runSequence('updateFormValueSequence', {
+        key: `statistics.${i}.year`,
+        value: 2019,
+      });
+
+      await test.runSequence('updateFormValueSequence', {
+        key: `statistics.${i}.deficiencyAmount`,
+        value: '$1,000',
```
Comment (rachaelparris) [inline web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js]: I think you _should_ get a validation error from this. The component I'm using actually calls `updateFormValueSequence` on the number without the $ and , in it.

File: web-client/integration-tests/petitionsClerkQCPetitionStatistics.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  petitionerChoosesCaseType(test);
+  petitionerCreatesNewCase(test, fakeFile);
+  petitionerViewsDashboard(test);
+
+  loginAs(test, 'petitionsclerk');
+  petitionsClerkViewsSectionDocumentQC(test);
+  petitionsClerkAssignsWorkItemToSelf(test);
+  petitionsClerkViewsMyDocumentQC(test);
+  petitionsClerkSelectsFirstPetitionOnMyDocumentQC(test);
+  petitionsClerkEditsPetitionInQCIRSNotice(test);
```
Comment (rachaelparris) [inline web-client/integration-tests/petitionsClerkQCPetitionStatistics.test.js:31]: Could you just add this to the `petitionsClerkPaperCaseJourney.test.js` or somewhere similar? That would avoid needing to do all the setup and case creation again to test just this piece.

Comment (matthopson) [reply web-client/integration-tests/petitionsClerkQCPetitionStatistics.test.js:31]: Sure can!

File: web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js (hunk 2 above)
Comment (matthopson) [reply web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js]: Ok. I'll try and write some more coverage around that.

File: web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js (hunk 1 above)
Comment (matthopson) [reply web-client/integration-tests/journey/petitionsClerkEditsPetitionInQCIRSNotice.js]: Kind-of. I intended to add to it (for the review screen), but I kinda wanted to do that in another PR. I'm going to need to pick this up in the morning anyway, so I may go ahead and include it with this.

**Review by codyseibert — APPROVED** [review]

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**